go_v2/main.go

```diff
@@ -0,0 +1,84 @@
+/*
```

Comment:
If we change the repo to a MIT license, can we take it out of the file as an include?